PRIVACY_POLICY.md

```diff
@@ -0,0 +1,38 @@
+<!--
+Copyright (c) 2020 The Gamepad Navigator Authors
+See the AUTHORS.md file at the top-level directory of this distribution and at
+https://github.com/fluid-lab/gamepad-navigator/raw/master/AUTHORS.md.
+
+Licensed under the BSD 3-Clause License. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the BSD 3-Clause License at
+https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
+-->
+
+# Privacy Policy
+
+The Gamepad Navigator allows the user to navigate Chromium-based browsers and web pages using a game controller. It
+provides various features that allow navigation using gamepad such as web page scrolling, tab navigation, click on
+buttons and hyperlinks, et cetera. Due to the nature of browser and web page navigation, the Gamepad Navigator requires
+permission to interact with the content in the active web pages, browser windows, and tabs.
+
+## Information the Gamepad Navigator collects
```

Comment:
What do their guidelines say about "collects"?  I'd like to see how they distinguish between the information you have access to and whether you store or transmit it.

Reply:
In general, the Chrome extension [privacy policy FAQ](https://developer.chrome.com/webstore/user_data) has stated a few cases about the information collection or "information handling", to be precise. You can read more about those cases [here](https://developer.chrome.com/webstore/user_data#ques_2). 

> Clipping or scraping content from a website that the user visits, such as taking screenshots or capturing data from a web page

This is one of the cases about information collection/handling that I copied from their FAQ, which comes close to what the Navigator does (even though the data is stored temporarily).

Reply:
Yes, at issue here is whether we "collect" anything at all or "handle" it, and their page is also vague on that.  Their examples do make it sound like the key concerns are around transmitting personal data of any kind.